### PR TITLE
test datasets that work on the whole pipeline

### PR DESCRIPTION
**Pull request by LaurenceKuhl**

### CODE REVIEW

File: testdata/samplesheet_test.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 sample,fastq_1,fastq_2,condition
-ERR376998,https://raw.githubusercontent.com/nf-core/test-datasets/crisprseq/testdata/ERR376998.small.fastq.gz,,control
-ERR376999,https://raw.githubusercontent.com/nf-core/test-datasets/crisprseq/testdata/ERR376999.small.fastq.gz,,treatment
\ No newline at end of file
+SRR8983579,/Users/laurencekuhlburger/Documents/crisprseq/nf-core-test/test-datasets/testdata/sub1.fq.gz,,control
+SRR8983580,/Users/laurencekuhlburger/Documents/crisprseq/nf-core-test/test-datasets/testdata/sub2.fq.gz,,treatment
```
Comment (mirpedrol) [inline testdata/samplesheet_test.csv:3]: There aren't files called `sub1.fq.gz` or `sub2.fq.gz`, shouldn't they be `SRR....small.fasq.gz`?

Comment (LaurenceKuhl) [reply testdata/samplesheet_test.csv]: hey sorry yes i forgot to save the file, should be good now

File: testdata/samplesheet_test.csv [hunk 2]
```diff
@@ -1,3 +1,3 @@
 sample,fastq_1,fastq_2,condition
-ERR376998,https://raw.githubusercontent.com/nf-core/test-datasets/crisprseq/testdata/ERR376998.small.fastq.gz,,control
-ERR376999,https://raw.githubusercontent.com/nf-core/test-datasets/crisprseq/testdata/ERR376999.small.fastq.gz,,treatment
\ No newline at end of file
+SRR8983579,https://raw.githubusercontent.com/nf-core/test-datasets/crisprseq/test-datasets/SRR8983579.small.fastq.gz,control
+SRR8983580,https://raw.githubusercontent.com/nf-core/test-datasets/crisprseq/test-datasets/SRR8983580.small.fastq.gz,,treatment
```
Comment (mirpedrol) [inline testdata/samplesheet_test.csv:3]:
```suggestion
SRR8983579,https://raw.githubusercontent.com/nf-core/test-datasets/crisprseq/testdata/SRR8983579.small.fastq.gz,,control
SRR8983580,https://raw.githubusercontent.com/nf-core/test-datasets/crisprseq/testdata/SRR8983580.small.fastq.gz,,treatment

```

**Review by mirpedrol — APPROVED** [review]
LGTM :)

**Pull request merged**